python/pyspark/sql/connect/streaming/readwriter.py

```diff
@@ -339,7 +342,9 @@ def table(self, tableName: str) -> "DataFrame":
 
 
 class DataStreamWriter:
-    def __init__(self, plan: "LogicalPlan", session: "SparkSession") -> None:
+    def __init__(
+        self, plan: "LogicalPlan", session: "SparkSession"
+    ) -> None:
```

Comment:
Not needed?

Reply:
hmmm not sure why is this, maybe the reformatter does that. I'll remove it.